packages/charts/src/components/chart-tooltip/chart-tooltip.component.tsx

```diff
@@ -65,6 +70,7 @@ const ChartTooltip: FC<Props> = ({ children, visible, x, y }) => {
             <Tooltip
               mode={theme.tooltip.mode}
               arrowDirection={arrowDirection as Position}
+              top={arrowOffset}
```

Comment:
I would consider re-naming that property to something more meaningful. As `top` suggest position of tooltip Y - but it's related to arrow.